erigon-lib/state/aggregator_bench_test.go

```diff
@@ -287,3 +287,15 @@ func Benchmark_Recsplit_Find_ExternalFile(b *testing.B) {
 		require.EqualValues(b, keys[p], key)
 	}
 }
+
+func BenchmarkAggregator_BeginFilesRo(b *testing.B) {
```

Comment:
This bench is ok. it testing `latency`. 
But we need one more bench to test `throughput`. 

Let me describe: 
- we know that `BeginRo` will be used in each RPC call and on each P2P message and in many background goroutines and in many other places (and remote and local).
- we know that there is mutex now 
- so, need bench for many parallel begins and rollback. see how in stdlib benchmarked mutex:
`BenchmarkMutexUncontended`, and neighbours. 
- we don't need test mutex perf itself. we just need understand current throughput of: `agg.BeginRo` and `temporalDB.BeginRo`. how much concurrent read transactions we can do. then in future we will take a look on bottlenecks.

Reply:
see the "throughput benchmark" comment above...it seems deterioration happens above 2^20 goroutines, which is about a million.